lib/events/dynamoevents/dynamoevents.go

```diff
@@ -134,6 +143,20 @@ func (cfg *Config) SetFromURL(in *url.URL) error {
 		cfg.Endpoint = endpoint
 	}
 
+	const boolErrorTemplate = "failed to parse URI %q flag %q - %q, supported values are 'true', 'false', or any other" +
+		"supported boolean in https://pkg.go.dev/strconv#ParseBool"
```

Comment:
I still think this can be much simpler.

Do we even need to raise an error here? `ParseBool` will return false on error, which is probably what you want anyway. Would it make sense to just log the warning that we're defaulting to false because we didn't understand the flag, and otherwise continue?